packages/@aws-cdk/aws-synthetics/lib/schedule.ts

```diff
@@ -64,6 +64,45 @@ export class Schedule {
     return new Schedule(`cron(${minute} ${hour} ${day} ${month} ${weekDay} ${year})`);
   }
 
+  /**
+   * Convert a Schedule expression's minutes in to a Duration.
+   *
+   * @param expression Schedule expression such as 'rate(2 minutes)'
+   */
+  public static expressionToRateDuration(expression: string): Duration {
+    if (expression.includes('cron')) {
```

Comment:
This is heckin' hacky, but I couldn't come up with a way around it aside from having `Schedule` store the input durations.